From f3eb62a3024611cf3df729eac967a777eabfcc79 Mon Sep 17 00:00:00 2001
From: Luca Boccassi <bluca@debian.org>
Date: Wed, 22 May 2024 21:17:24 +0100
Subject: [PATCH] Update autosuspend hwdb

ninja -C build update-hwdb-autosuspend

(cherry picked from commit 55435a9eb21c40ed6bdb847f63e178596edb6985)

Related: RHEL-5950
---
 hwdb.d/60-autosuspend-fingerprint-reader.hwdb | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/hwdb.d/60-autosuspend-fingerprint-reader.hwdb b/hwdb.d/60-autosuspend-fingerprint-reader.hwdb
index dc474c5e7c..ea656123f9 100644
--- a/hwdb.d/60-autosuspend-fingerprint-reader.hwdb
+++ b/hwdb.d/60-autosuspend-fingerprint-reader.hwdb
@@ -359,6 +359,12 @@ usb:v0A5Cp5842*
 usb:v0A5Cp5843*
 usb:v0A5Cp5844*
 usb:v0A5Cp5845*
+usb:v0A5Cp5860*
+usb:v0A5Cp5863*
+usb:v0A5Cp5864*
+usb:v0A5Cp5865*
+usb:v0A5Cp5866*
+usb:v0A5Cp5867*
 usb:v0BDAp5812*
 usb:v10A5p0007*
 usb:v10A5p9200*
